src/app/components/Backbutton/Backbutton.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import BackIcon from '../assets/BackIcon';
+import styles from './Backbutton.module.css';
+
+type BackbuttonProps = {
+  className?: string;
+};
+
+function Backbutton({ className }: BackbuttonProps): JSX.Element {
+  return (
+    <button className={`${styles.button} ${className}`}>
+      <BackIcon width="0.9rem" height="1.5rem" />
+    </button>
```

Comment:
Also needs an `onClick` prop

Reply:
Could you please check if this is the correct solution